### PR TITLE
[SPARK-28083][SQL][TEST][FOLLOW-UP] Enable LIKE ... ESCAPE test cases

### PR DESCRIPTION
**Pull request by beliefer**
### What changes were proposed in this pull request?
This PR is a follow-up to https://github.com/apache/spark/pull/25001

### Why are the changes needed?
No

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
Pass the Jenkins with the newly update test files.

### CODE REVIEW

File: sql/core/src/test/resources/sql-tests/inputs/postgreSQL/strings.sql [hunk 1]
```diff
@@ -272,53 +272,53 @@ SELECT 'indio' NOT LIKE 'in_o' AS `true`;
 
 -- [SPARK-28083] ANSI SQL: LIKE predicate: ESCAPE clause
```
Comment (dongjoon-hyun) [inline sql/core/src/test/resources/sql-tests/inputs/postgreSQL/strings.sql]: Could you remove this comment?

**Review by maropu — APPROVED** [review]
LGTM except for the @dongjoon-hyun comment.

File: sql/core/src/test/resources/sql-tests/inputs/postgreSQL/strings.sql (hunk 1 above)
Comment (beliefer) [reply sql/core/src/test/resources/sql-tests/inputs/postgreSQL/strings.sql]: OK

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM. Merged to master.
Thank you, @beliefer and @maropu .

**Comment by beliefer** [comment]
@dongjoon-hyun @maropu Thanks!

**Pull request closed without merging**